test/scenarios/activegate/default/default_test.go

```diff
@@ -1,6 +1,6 @@
 //go:build e2e
 
-package _default
+package activegatebasic
```

Comment:
this was just renamed 😅

Reply:
yeah please rename it back to _default

Reply:
hmm sorry must be some problems with merge 😞